lighthouse-cli/chrome-launcher.ts

```diff
@@ -214,11 +214,13 @@ class ChromeLauncher {
         });
 
         log.log('ChromeLauncher', 'Killing all Chrome Instances');
-        this.chrome.kill();
-
         if (process.platform === 'win32') {
           spawnSync(`taskkill /pid ${this.chrome.pid} /T /F`);
+        } else {
+          process.kill(-this.chrome.pid);
```

Comment:
If we launched Chrome we want to kill it, even in the case of error. For some reason, when the error is that we can't connect to Chrome (only seen on Travis so far and fixed in #922), `this.chrome.kill()` doesn't work and the `close` event never fires. Killing the process group does work, however, and doesn't appear to have any downsides (please chime in if you know any I'm missing).

(it seems Windows [will throw when trying to kill the process group](https://nodejs.org/api/process.html#process_process_kill_pid_signal), but since we're using `taskkill` to kill on windows anyway, it's easy to just add this to the `else` clause)